builtin/logical/cassandra/path_roles.go

```diff
@@ -131,6 +131,9 @@ func (b *backend) pathRoleCreate(
 	}
 
 	consistencyStr := data.Get("consistency").(string)
+	if consistencyStr == "" {
```

Comment:
This should be the value of the "Default" member of the field. That way `Get` will automatically populate it with `"Quorum"` if they do not specify a different value.